### PR TITLE
Add New rocket to process batch files

### PR DESCRIPTION
**Pull request by rdoria1**
## Description
Add new Rocket:
- Create new source Bucket with event notifications.
- Create new Lambda (triggered by event notification) that will persist split the file with a given N size lines passed in the params it will also persist a new event for each new file added
- These new files will be persisted in a new Staging bucket

## Changes
Add new package for new rocket + upadte lerna.json and packge.json with new package.

## Checks
- [X] Project Builds
- [X] Project passes tests and checks
- [ ] Updated documentation accordingly

### CODE REVIEW

**Review by javiertoledo — APPROVED** [review]
LGTM, but if we are planning to maintain this rocket, we'll need some documentation, and unit/integration tests.

**Comment by rdoria1** [comment]
> LGTM, but if we are planning to maintain this rocket, we'll need some documentation, and unit/integration tests.

@javiertoledo Yes I think we are planning to maintain this rocket. I will add doc and tests :)

File: packages/rocket-batch-file-process-aws-infrastructure/package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@boostercloud/rocket-batch-file-process-aws-infrastructure",
+  "version": "0.10.0",
```
Comment (charlietfe) [inline packages/rocket-batch-file-process-aws-infrastructure/package.json:3]:
```suggestion
  "version": "0.11.2",
```

**Review by charlietfe — APPROVED** [review]
LGTM, I left some minor comments which don't need to take any action. Regarding tests, I could handle it ;-)

File: packages/rocket-batch-file-process-aws-infrastructure/src/lambdas/index.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        Body: content.join('\n'),
+      })
+      .promise()
+  } catch (e) {
+    console.log('[ROCKET#batch-file] An error occurred while performing a PutObject operation: ', e)
+  }
+}
+
+async function saveEvent(key: string): Promise<void> {
+  const { v4: uuidv4 } = require('uuid')
```
[the remark this answers is not in the document]
Comment (rdoria1) [reply packages/rocket-batch-file-process-aws-infrastructure/src/lambdas/index.ts:75]: I would need to import the framework-types as dependency, I wanted to keep the lambda as plain as possible hehe

**Pull request merged**